### PR TITLE
AOM-125: Clean up codebase by enforcing consistent code style guidelines

### PR DESCRIPTION
**Pull request by Charpell**
## JIRA TICKET NAME
[AOM-125: Clean up codebase by enforcing consistent code style guidelines](https://issues.openmrs.org/browse/AOM-125)

### SUMMARY
Up until now Addon Manager has been written by a dozen of programmers over several months, and on code review dose not use any standard code style guidelines, which means the code isn't as readable or clean as it should be.

This PR aims to clean up the codebase.

### CODE REVIEW

**Review by segunolalive — COMMENTED** [review]
LGTM

**Comment by justmesam** [comment]
Looks good

File: app/js/components/manageApps/ManageSettings.jsx [hunk 1]
```diff
@@ -87,8 +87,9 @@ export default class ManageSettings extends React.Component {
     let {
       AppBaseURLValue,
       AppFolderPathValue,
-      AppStoreURLValue
+      AppStoreURLValue,
```
Comment (dkayiwa) [inline app/js/components/manageApps/ManageSettings.jsx:90]: Is the above comma needed?

Comment (Charpell) [reply app/js/components/manageApps/ManageSettings.jsx:90]: Yes there is always a comma at the end of each object literal

**Pull request merged**